data/json/items/ammo/nail.json

```diff
@@ -8,7 +8,7 @@
     "material": [ "steel" ],
     "symbol": "=",
     "color": "dark_gray",
-    "proportional": { "price": 3, "damage": { "damage_type": "stab", "amount": 3.0, "armor_penetration": 3.0 } }
```

Comment:
This and the one below it should do bullet, not stab.
Also, I think the nail should be worse as ammo, but I'm not a stickler on that (3 armor pen and 4 damage still seems a bit high, and dispersion seems a little low).

Reply:
Done, both do bullet damage now.
Coilgun with nails now does 1+3 damage, 1 armor pen, 180+180 dispersion.
Coilgun with flechettes now does 1+6 damage, 3 armor pen, 180+90 dispersion.